compiler/rustc_middle/src/hir/map/mod.rs

```diff
@@ -895,7 +891,7 @@ impl<'hir> Map<'hir> {
 
     #[inline]
     fn opt_ident(self, id: HirId) -> Option<Ident> {
-        match self.get(id) {
+        match self.find(id)? {
```

Comment:
What codepath triggers this specifically? Just curious.

Reply:
`def_span` for anonymous constants without HIR.

Reply:
Should we keep a FIXME here too?

Reply:
Not all passed `HirId`s are required have HIR nodes or `Ident`s, so the new behavior looks reasonable to me, we just didn't hit this case before.